autoload/ale/fixers/xmllint.vim

```diff
@@ -0,0 +1,27 @@
+" Author: Cyril Roelandt <tipecaml@gmail.com>
+" Description: Integration of xmllint with ALE.
+
+call ale#Set('xml_xmllint_executable', 'xmllint')
+call ale#Set('xml_xmllint_options', '')
+call ale#Set('xml_xmllint_indentsize', 2)
+
+function! ale#fixers#xmllint#Fix(buffer) abort
+    let l:executable = ale#Escape(ale#Var(a:buffer, 'xml_xmllint_executable'))
+    let l:filename = ale#Escape(bufname(a:buffer))
+    let l:command = l:executable . ' --format ' . l:filename
+
+    let l:indent = ale#Var(a:buffer, 'xml_xmllint_indentsize')
+    if l:indent isnot# ''
+        let l:env = 'XMLLINT_INDENT="' . repeat(' ', l:indent) . '"'
```

Comment:
Use `ale#Env('XMLLINT_INDENT', repeat(' ', l:indent))` instead. The result will work on Windows or Unix systems.